extensions/stats/testdata/istio/metadata-exchange_with_wasm_filter.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: metadata-exchange
+spec:
+  filters:
+  - filterConfig:
+      configuration: envoy.wasm.metadata_exchange
```

Comment:
I don't think we need `configuration` value here, do we?

Reply:
As per @mandarjog , looks like its needed